app/components/Contacts/Network/Network.js

```diff
@@ -306,7 +259,7 @@ class Network extends Component {
                             onClick={() =>
                               blockExplorer.showTransaction(
                                 networkInfo,
-                                channelObj.closing_txid || channel.channel_point.split(':')[0]
+                                closing_txid || channel_point.split(':')[0]
```

Comment:
I suppose `channel_point` can't be null? I see it's also used as a key

Reply:
Yes - following on from our discussion the other day about trying to identify a unique key for the channels, I found that whilst `chan_id` is not always set, `channel_point` and I believe it to be unique. From the docs:


```
The outpoint (txid:index) of the funding transaction. With this value, Bob will be able to generate a signature for Alice’s version of the commitment transaction.
```

```
funding_txid_bytes | bytes | Txid of the funding transaction
funding_txid_str | string | Hex-encoded string representing the funding transaction
output_index | uint32 | The index of the output of the funding transaction
````
https://api.lightning.community/#channelpoint

This identifier is used to uniquely identify channels for several operations in the lnd api.